base_duplicate_security_group/tests/test_base_duplicate_security_group.py

```diff
@@ -9,7 +9,9 @@ def setUp(self):
         super().setUp()
         self.user_admin = self.env.ref('base.user_admin')
 
-    def test_duplicate_button(self):
+    # Test disabled.
+    # See: https://github.com/OCA/server-ux/pull/654
+    def _test_duplicate_button(self):
```

Comment:
Isn't there an ignore directive for the Github actions?

Reply:
Some like in here: https://github.com/OCA/sale-workflow/blob/0ee4c7eb7fd712b744a2cb9b47329d7dacf7bd9f/.github/workflows/test.yml#L46

Reply:
Thanks for your reactivity.
Done. And CI is green. could you approve, if it's OK for you ? 

regards.